core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -319,13 +319,24 @@ public Options fetch(ColumnType... colsToFetch) {
       return this;
     }
 
+    /**
+     * Set the range to the table range that stores the provided level's metadata. For example
+     * {@link DataLevel#USER} will set the range to the metadata table. {@link DataLevel#METADATA}
+     * will set the range to the root table.
+     */
     @Override
     public Options forLevel(DataLevel level) {
       this.level = level;
       this.range = TabletsSection.getRange();
       return this;
     }
 
+    /**
+     * Set the range to the data level that contains the metadata references for the provided
+     * TableId. For example, if the TableId is the metadata table id, then the table range will be
+     * root metadata that points to the metadata. Likewise, if the TableId is a user table, then the
```

Comment:
This comment is a bit hard to follow. 

Can the metadata table location references be simplified
like `root metadata range` vs `root metadata that points to the metadata`

Reply:
I'm open to suggestions.  I was trying to convey that user -> metadata, metadata -> root, and root -> ZooKeeper.  One issue is that metadata is overloaded.  When we say metadata, is it the table? The information for a level that references point to the next level?  Even that last sentence is awkward for me, so if someone has any suggestions I'll happily hit commit suggestion and move on.